conjur-oss/README.md

```diff
@@ -186,15 +187,44 @@ $  helm install \
    https://github.com/cyberark/conjur-oss-helm-chart/releases/download/v<VERSION>/conjur-oss-<VERSION>.tgz
 ```
 
-*NOTE:* If using the Kubernetes authenticator for Conjur, the `account` value
+*NOTE:* If using the Kubernetes authenticator for Conjur, the `account.name` value
 (see [Configuration](#configuration)) must match the initial Conjur account
-created. For example, given the following command:
+created.
+
+### Configuring Conjur Accounts
+
+By setting `account.create` to `true`, you can direct your Conjur
+container to create an account during startup. To retrieve the credentials
+for this account, perform the following commands:
 
 ```sh-session
-$  kubectl exec -n "$CONJUR_NAMESPACE" "$POD_NAME" --container=conjur-oss conjurctl account create default
+ACCOUNT_NAME=<conjur-account-name>
+CONJUR_NAMESPACE=<conjur-namespace>
+HELM_RELEASE=<helm-release>
+POD_NAME=$(kubectl get pods --namespace "$CONJUR_NAMESPACE" \
+            -l "app=conjur-oss,release=$HELM_RELEASE" \
+            -o jsonpath="{.items[0].metadata.name}")
+kubectl exec --namespace "$CONJUR_NAMESPACE" \
+            "$POD_NAME" \
+            --container=conjur-oss \
+            -- conjurctl role retrieve-key "$ACCOUNT_NAME":user:admin
 ```
 
-The chart value for `account` would be expected to equal `default`.
+If you set `account.create` to `false`, or did not provide a value, an admin account will
+need to be created. To create an account, use the following commands:
+
+```sh-session
+ACCOUNT_NAME=<Name for Conjur account to be created>
+POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} \
```

Comment:
Oops, this section needs to have the Helm template language stuff replaced
(e.g. `{{ .Release.Namespace }}`, `{{ .Release.Name }}`, etc).